components/rdv/DeleteRdvModal.tsx

```diff
@@ -7,23 +7,32 @@ interface DeleteRdvModalProps {
   aDesJeunesDUnAutrePortefeuille: boolean
   onClose: () => void
   performDelete: () => Promise<void>
+  evenementTypeAC: boolean
 }
 
 export default function DeleteRdvModal({
   aDesJeunesDUnAutrePortefeuille,
   onClose,
   performDelete,
+  evenementTypeAC,
 }: DeleteRdvModalProps) {
   function handleCloseModal() {
     onClose()
   }
 
   const message = aDesJeunesDUnAutrePortefeuille
-    ? 'Vous allez supprimer un événement qui concerne des jeunes qui ne sont pas dans votre portefeuille'
+    ? `Vous allez supprimer ${
+        evenementTypeAC ? 'une animation collective' : 'un rendez-vous'
+      } événement qui concerne des jeunes qui ne sont pas dans votre portefeuille`
     : 'L’ensemble des bénéficiaires sera notifié de la suppression'
 
   return (
-    <Modal title='Suppression de l’événement' onClose={handleCloseModal}>
+    <Modal
+      title={`Suppression ${
+        evenementTypeAC ? 'de l‘animation collective' : 'du rendez-vous'
```

Comment:
```suggestion
        evenementTypeAC ? 'de l’animation collective' : 'du rendez-vous'
```
c'est subtil, mais il faut changer le caractère utilisé pour l’apostrophe, elle est à l'envers.
Sur mac pour avoir la vraie apostrophe française, `option shift 4`
https://unicode-table.com/fr/2019/
à répercuter partout où tu as mis une apostrophe :-)
![Capture d’écran 2023-02-07 à 15 57 50](https://user-images.githubusercontent.com/62615060/217280323-955df04c-e17b-4847-a5d5-fc8a56c6f2ab.png)